scripts/diff_stix.py

```diff
@@ -420,19 +429,70 @@ def placard(item):
                     return f"[{item['name']}]({self.site_prefix}/{self.getUrlFromStix(item, is_subtechnique)})"
 
 
+            def change_details_placard(item, tab_count = 2):
+                attack_id = item['external_references'][0]['external_id']
+                details = self.change_details[attack_id]
+                changes = []
+                for change in details:
+                    what_changed, index_changed = (f"{change[1][0]}", change[1][1]) if type(change[1]) == list else (change[1], None)
+                    if what_changed not in self.no_details:
+                        self.change_details["num_changes"] += 1
+                        how_changed = change[2]
+                        change_type = change[0]
+                        indent = "\t" * tab_count
+                        if change_type == "add":
+                            changes.append(f"{indent}* Added entries to {what_changed} field")
```

Comment:
This doesn't seem to always work -- what_changed is empty in some cases leading to a double space in the output:

```
	* Added entries to  field
		* x_mitre_is_subtechnique

		```
		False
		```

	* Removed entries from  field
		* x_mitre_contributors:  ['Itamar Mizrahi', 'Tristan Bennett, Seamless Intelligence']
```
I'm not sure exactly what the cause is.